0101-nix formatting.md

```diff
@@ -0,0 +1,65 @@
+---
+feature: nix_formatting
+start-date: 2021-08-17
+author: Raphael Megzari
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: Timothy DeHerrera (nrdxp), 0x4A6F
+shepherd-leader: Jonas Chevalier (zimbatm)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+
+[summary]: #summary
+
+Decide on a recommended automated formatter for nix files in nixpkgs.
+
+# Motivation
+
+[motivation]: #motivation
+
+Prevent debate around how things should be formatted.
+
+# Detailed design
+
+[design]: #detailed-design
+
+The implementation of this RFC should include several parts
+
+- Agree on an automated formatter. [nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt) is proposed here.
+- If adopted, agree on a schedule to format nixpkgs codebase. Should it be done at once, or on a per package basis.
+- Potentially agree on a hook to enforce formatting
+
```

Comment:
I think a few formatting choices still need to be discussed. One of the largest issues is probably where the trailing commands in attrsets should be https://github.com/nix-community/nixpkgs-fmt/issues/248. The main arguments have probably already been said in that issue - I don't know if we should continue the discussion here or there.

Reply:
I left a comment in the issue